testsuite/MDAnalysisTests/utils/test_streamio.py

```diff
@@ -213,11 +217,9 @@ def test_expanduser_noexpansion_returns_NamedStream(self):
                      err_msg=("os.path.expanduser() without '~' did not "
                               "return NamedStream --- weird!!"))
 
-    # expandvars(NamedStream) does not work interactively, so it is a knownfailure
-    # Segmentation fault when run as a test on Mac OS X 10.6, Py 2.7.11 [orbeckst]
-    @dec.skipif(True)
-    @dec.skipif("HOME" not in os.environ)
-    @knownfailure
+    @pytest.mark.skipif(True, reason='Segmentation fault when run as a test on Mac OS X 10.6, Py 2.7.11 [orbeckst]')
```

Comment:
does this fail on travis when you remove this line? The comment refers to a 6 year old OS.